__test__/unit/blockchain_api/error_scenario_runner.ts

```diff
@@ -51,38 +69,159 @@ export class ErrorScenarioRunner {
         }
 
         // Construct all necessary dependencies.
-        const contractsAPI = new ContractsAPI(this.web3);
-        const blockchainAPI = new BlockchainAPI(this.web3, contractsAPI);
-        const signerAPI = new SignerAPI(this.web3, contractsAPI);
+
+        this.contractsAPI = new ContractsAPI(this.web3);
+        this.blockchainAPI = new BlockchainAPI(this.web3, this.contractsAPI);
+        this.signerAPI = new SignerAPI(this.web3, this.contractsAPI);
+        this.orderAPI = new OrderAPI(this.web3, this.contractsAPI);
+
         const {
             debtKernel,
             debtRegistry,
             debtToken,
```

Comment:
Neither `debtRegistry` nor `debtToken` are used in this class.  Please remove from the destructuring.

Reply:
Good catch